official/docs/java/endShipper/UpdateEndShipper.java

```diff
@@ -0,0 +1,27 @@
+import com.easypost.EasyPost;
+import java.util.HashMap;
+
+public class UpdateEndShipper {
+    public static void main(String[] args) {
+        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
+
+        EndShipper endShipper = EndShipper.retrieve("es_...");
+
+        Map<String, Object> updateParams = new HashMap<String, Object>();
```

Comment:
```suggestion
        HashMap<String, Object> updateParams = new HashMap<String, Object>();
```
This will need to be `HashMap` now